modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_problem/sweep_chunks/aah_sweep_chunk_rz.cc

```diff
@@ -1,17 +1,17 @@
 // SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
 // SPDX-License-Identifier: MIT
 
-#include "modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_solver/sweep_chunks/lbs_curvilinear_sweep_chunk_pwl.h"
-#include "modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/fluds/aah_fluds.h"
-#include "modules/linear_boltzmann_solvers/lbs_solver/groupset/lbs_groupset.h"
+#include "modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_problem/sweep_chunks/aah_sweep_chunk_rz.h"
+#include "modules/linear_boltzmann_solvers/discrete_ordinates_problem/sweep/fluds/aah_fluds.h"
+#include "modules/linear_boltzmann_solvers/lbs_problem/groupset/lbs_groupset.h"
 #include "framework/math/spatial_discretization/spatial_discretization.h"
 #include "framework/math/quadratures/angular/curvilinear_product_quadrature.h"
 #include "framework/mesh/mesh_continuum/mesh_continuum.h"
 
 namespace opensn
 {
 
-SweepChunkPwlrz::SweepChunkPwlrz(const std::shared_ptr<MeshContinuum> grid,
+AahSweepChunkRZ::AahSweepChunkRZ(const std::shared_ptr<MeshContinuum> grid,
```

Comment:
i guess this is the name convention, and we cannot use AAH (AdamsAdamsHawkins)

Reply:
This matches the naming scheme of the other sweep chunks `AahSweepChunk` and `CbcSweepChunk`.
However, SPDSs use `AAH_SPDS` and `CBC_SPDS` 🤔